cmd/apiserver-watcher/iptables

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+exec chroot /rootfs /usr/sbin/iptables "$@"
```

Comment:
Question: what is this binary being used for? Does go-iptables make direct calls to the binary?

Reply:
right, but the binary use some shared libraries, so if the call is not chrooted it fails

Reply:
FYI, we also need to know if iptables-nft or iptables-legacy is in use. And we might need to `modprobe`

Reply:
This is the same as in openshift/sdn https://github.com/openshift/sdn/blob/master/images/iptables-scripts/ip6tables
I think that OS symlinks /usr/sbin iptables, and since this pods mounts the whole rootfs under /rootfs doing a chroot will work with whatever the OS has configured.
Modprobe is done by the iptables userspace tools IIRC